Intersect (Core)/Config/PartyOptions.cs

```diff
@@ -14,5 +14,11 @@ public class PartyOptions
 
         public int NpcDeathCommonEventStartRange = 0;
 
+        /// <summary>
+        /// It will determine if you will have an XP bonus for each member, Eg 10, will give 10% more to the monster's total XP, then split equally to all members.
+        /// A monster that would give 100 XP, now gives 120, getting 60 for each member, when 2 in the party.
+        /// </summary>
+        public float BonifyExpPercentPerMember = 0;
```

Comment:
Shouldn't this be called something like "BonusExperiencePercentPerMember” for clarity?